images/manageiq-ui-worker/Dockerfile

```diff
@@ -10,7 +10,7 @@ LABEL name="manageiq-ui-worker" \
       summary="ManageIQ user interface worker image"
 
 # Install httpd and remove the existing httpd config from manageiq-appliance
-RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install httpd && \
+RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install httpd python2 && \
```

Comment:
I would expect that we already have a python installed. Maybe we can set the `PYTHON` env var instead of installing an extra python version as mentioned in the error message `gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.`

Reply:
so @carbonin I tried that but we have only python3 already installed and seems it specifically needs python2, this is the error I get if I force it to use pre-installed python3
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` failed Error: not found: python2
gyp verb `which` failed     at getNotFoundError (/usr/share/gems/bundler/gems/manageiq-ui-classic-165afa406837/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/usr/share/gems/bundler/gems/manageiq-ui-classic-165afa406837/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/usr/share/gems/bundler/gems/manageiq-ui-classic-165afa406837/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /usr/share/gems/bundler/gems/manageiq-ui-classic-165afa406837/node_modules/which/which.js:89:16
gyp verb `which` failed     at /usr/share/gems/bundler/gems/manageiq-ui-classic-165afa406837/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /usr/share/gems/bundler/gems/manageiq-ui-classic-165afa406837/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:166:21)
gyp verb `which` failed  python2 Error: not found: python2

Reply:
also please refer to the finding here https://github.com/ManageIQ/manageiq-pods/issues/460#issuecomment-619064709

Reply:
Okay, can we make this conditional on arch then? I don't want extra python runtimes cluttering things up if we don't need them

Reply:
@carbonin  thanks for the feedback. Done - please review, thank you!